src/main/scala/cosas/ops/typeSets/Conversions.scala

```diff
@@ -107,88 +107,3 @@ object ToMap {
     def apply(s: D :~: T): Out =  rest(s.tail) + (key -> s.head.value)
```

Comment:
mmm. Let me add generic serializers/parsers to the value using the type label and see if it works